<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myMainWindow</class>
 <widget class="QMainWindow" name="myMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TableView</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>PlainTextEdit</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="actModelData"/>
   <addaction name="separator"/>
   <addaction name="actAppend"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actAlignLeft"/>
   <addaction name="actAlignCenter"/>
   <addaction name="actAlignRight"/>
   <addaction name="separator"/>
   <addaction name="actFontBold"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/open.jfif</normaloff>:/img/image/open.jfif</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/save.jfif</normaloff>:/img/image/save.jfif</iconset>
   </property>
   <property name="text">
    <string>另存文件</string>
   </property>
  </action>
  <action name="actAppend">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/boy.jfif</normaloff>:/img/image/boy.jfif</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/girl.jfif</normaloff>:/img/image/girl.jfif</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/clear.jfif</normaloff>:/img/image/clear.jfif</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/close.jfif</normaloff>:/img/image/close.jfif</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actModelData">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/toolbarlab.jfif</normaloff>:/img/image/toolbarlab.jfif</iconset>
   </property>
   <property name="text">
    <string>模型数据预览</string>
   </property>
  </action>
  <action name="actAlignLeft">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/standleft.jpg</normaloff>:/img/image/standleft.jpg</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
  </action>
  <action name="actAlignCenter">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/standmiddle.jpg</normaloff>:/img/image/standmiddle.jpg</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="actAlignRight">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/standright.jfif</normaloff>:/img/image/standright.jfif</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/image/fontbold.jfif</normaloff>:/img/image/fontbold.jfif</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
